<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fuse_calc</class>
 <widget class="QMainWindow" name="fuse_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATMega Fuse Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabFuseCalc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>371</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabFuseHex">
     <attribute name="title">
      <string>Fuse Hex</string>
     </attribute>
     <widget class="QComboBox" name="cmbSelectChip">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSelectChip">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Chip</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="grpFuseHex">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>171</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Hex Editor</string>
      </property>
      <widget class="QLineEdit" name="txtEFUSE">
       <property name="geometry">
        <rect>
         <x>82</x>
         <y>20</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtHFUSE">
       <property name="geometry">
        <rect>
         <x>82</x>
         <y>50</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtLFUSE">
       <property name="geometry">
        <rect>
         <x>82</x>
         <y>80</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnFuseHexDefault">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>61</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblFuseHexEFUSE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>HFUSE</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblFuseHexHFUSE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>HFUSE</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblFuseHexLFUSE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>LFUSE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnFuseHexApply">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>61</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFuseBit">
     <attribute name="title">
      <string>Fuse Bit</string>
     </attribute>
     <widget class="QGroupBox" name="grpFuse_0">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>101</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>LFUSE</string>
      </property>
      <widget class="QCheckBox" name="chkFUSE0_0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE1_0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE2_0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE3_0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE4_0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE5_0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE6_0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE7_0">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpFuse_1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>101</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>HFUSE</string>
      </property>
      <widget class="QCheckBox" name="chkFUSE0_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE1_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE2_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE3_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE4_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE5_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE6_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE7_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpFuse_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>101</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>EFUSE</string>
      </property>
      <widget class="QCheckBox" name="chkFUSE0_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE2_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE3_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE4_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE5_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE6_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkFUSE7_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>UNUSED</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="lblBit_0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bit 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBit_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bit 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bit 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bit 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBit_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bit 4</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBit_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bit 5</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBit_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bit 6</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBit_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bit 7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFuseBitDefault">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFuseBitApply">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Calc</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFuseBitUnrestrict">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Unrestrict</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabBOD">
     <attribute name="title">
      <string>Brown-Out</string>
     </attribute>
     <widget class="QComboBox" name="cmbBOD">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBODApply">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>85</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Calc</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabBootSz">
     <attribute name="title">
      <string>Boot Size</string>
     </attribute>
     <widget class="QComboBox" name="cmbBootSz">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBootSzApply">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>85</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Calc</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabOsc">
     <attribute name="title">
      <string>Oscillator</string>
     </attribute>
     <widget class="QPushButton" name="btnOscApply">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>85</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Calc</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbOsc">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>413</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
